README.md

```diff
@@ -71,7 +72,27 @@ By default:
 - The jnilib native files will be installed at: ``/Applications/HDFView.app/Contents//Resources/lib/``  This location can be used in the instructions below.
 - The JAR files will be installed in ``/Applications/HDFView.app/Contents/Java/``.
 
-#### Other from binaries
+
```

Comment:
In the section above, the step "2. Install the binary." is too vague. Does this mean there is an auto installer? In that case, say so. (eg "Launch the installer and follow on-screen instructions")